spring-ai-alibaba-agent-framework/src/main/java/com/alibaba/cloud/ai/graph/agent/DefaultBuilder.java

```diff
@@ -51,7 +51,7 @@ public ReactAgent build() {
 		if (chatClient == null) {
 
 			ChatClient.Builder clientBuilder = ChatClient.builder(model, this.observationRegistry == null ? ObservationRegistry.NOOP : this.observationRegistry,
-					this.customObservationConvention);
+					this.customObservationConvention, null);
```

Comment:
I have confirmed that `null` advisor observation will default to DEFAULT_OBSERVATION_CONVENTION. It's good to go with null.

https://github.com/spring-projects/spring-ai/blob/main/spring-ai-client-chat/src/main/java/org/springframework/ai/chat/client/advisor/DefaultAroundAdvisorChain.java#L86